src/view_page.cpp

```diff
@@ -324,10 +324,71 @@ void View::DrawStaffGrp(
     // for the bottom position we need to take into account the number of lines and the staff size
     int yBottom
         = last->GetDrawingY() - (lastDef->GetLines() - 1) * m_doc->GetDrawingDoubleUnit(last->m_drawingStaffSize);
-    int barLineWidth = m_doc->GetDrawingBarLineWidth(100);
+
+    int barLineWidth;
+    if (m_doc->GetType() == Facs) {
+        barLineWidth = m_doc->GetDrawingBarLineWidth(last->m_drawingStaffSize);
+    }
+    else {
+        barLineWidth = m_doc->GetDrawingBarLineWidth(100);
+    }
 
     // adjust the top and bottom according to staffline width
     x += barLineWidth / 2;
+    // yTop += m_doc->GetDrawingStaffLineWidth(last->m_drawingStaffSize) / 1;
```

Comment:
Why is this all new? This does not look like neume-specific

Reply:
I think it's just something that happened because of a quirk of git. It isn't new, but it might have ended up marked that way after a merge.

Reply:
This file seems to be out of sync. See for example `View::DrawBarLine`. I pushed the result of the merge (together with a few fixes to the XCode project) here https://github.com/rism-ch/verovio/tree/develop-merge-neume
It crashes on the CMN in DrawBarline. Could you double check what happened with this file?

Reply:
Once I finish the issue with unused glyphs I'll try and figure out what happened here.

Reply:
Seems to be good in 15aa883 in that `View::DrawBarLine` does not cause Verovio to crash for me in CMN. Can you confirm that this file seems to be fixed?